packages/react/src/listbox-item/ListboxItem.css.ts

```diff
@@ -73,6 +73,10 @@ export const item = recipe({
 
 export const title = recipe({
   base: [
+    {
+      alignItems: "center",
```

Comment:
we need `flexDirection: row` here if we want to re-use `Flex` component

maybe add a story so it's visually clear what's happening here

Reply:
Addressed vai.